src/module.d.ts

```diff
@@ -34,3 +34,74 @@ declare module '@twilio/cli-core' {
 
   export { baseCommands, services };
 }
+
+declare module '@twilio/cli-test' {
+  import { baseCommands } from '@twilio/cli-core';
+  import { Logger } from 'flex-plugins-utils-logger';
+  import {Context, Plugins} from "fancy-test/lib/types";
```

Comment:
linter doesn't complain about this line??

Reply:
I've asked it not to lint `.d.ts` there were other issues linting these types of files.